tests/orquestra/vqa/cost_function/cost_function_test.py

```diff
@@ -29,7 +29,7 @@
 
 RNGSEED = 1234
 
-BACKEND = SymbolicSimulator()
+SIMULATOR = SymbolicSimulator()
```

Comment:
I think you want to have this as a fixture. I think it's safer, cause objects might have state and I'm not 100% sure if defining it like this would make the state to persist between tests.
Doesn't matter in the case of this particular I think, cause Simulator is stateless, but just so you know.

Reply:
And yes, I know it was like this, you just refactored it ;)